index.html

```diff
@@ -209,54 +209,53 @@ <h2 class="speaker-header">featured speakers</h2>
         <button class="more">
            MORE <span class="fa-more"><i class="fa-solid fa-angle-down"></i></span>
         </button>
-    </div>
-    
+    </div> 
   </section>
- <section class="forum-partners">
```

Comment:
- [ ] `required`
- [ ] please add the required background
- [ ] please see screenshot below

`your template`
![image](https://user-images.githubusercontent.com/66230306/218248171-320aa0d1-01e9-4b00-a3dd-00d414991bc8.png)

`cindy shin template`
![image](https://user-images.githubusercontent.com/66230306/218248207-0af758cd-a37e-4453-ac17-2f7741776c8f.png)